cfgov/v1/atomic_elements/organisms.py

```diff
@@ -291,6 +291,25 @@ def field(self):
         widget = AtomicTableInput(table_options=self.table_options)
         return forms.CharField(widget=widget, **self.field_options)
 
+    def to_python(self, value):
+        new_value = super(AtomicTableBlock, self).to_python(value)
+        if new_value:
+            new_value['has_data'] = self.get_has_data(new_value)
+        return new_value
+
+    def get_has_data(self, value):
+        has_data = False
+        if value:
+            first_row_index = 1 if value['first_row_is_table_header'] else 0
+            first_col_index = 1 if value['first_col_is_header'] else 0
+
+            for row in value['data'][first_row_index:]:
+                for cell in row[first_col_index:]:
+                    if cell:
+                        has_data = True
+                        break
```

Comment:
Will checking `value['data'][1]` cause an exception if there isn't any data before the first header row? It might be nice to have some backend unit tests around this; I can help if it'd be useful.